receiver/zookeeperreceiver/documentation.md

```diff
@@ -38,6 +38,4 @@ metrics:
 | Name | Description |
 | ---- | ----------- |
 | direction | State of a packet based on io direction. |
-| server.state | State of the Zookeeper server (leader, standalone or follower). |
```

Comment:
It looks like the documentation generator needs updating here.

Reply:
done